style.css

```diff
@@ -0,0 +1,361 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	width: 100%;
+	max-width: 5000px;
+	margin: 20x auto; 
+}
+
+.navbar {
+	display: flex;
+	justify-content: flex-start;
+	gap: 50px;
+	padding: 15px;
+	padding-left: 45px;
+	background-color: #9a7b5c;
+	font-family: 'Roboto Flex', sans-serif;
+	font-size: 20px;
+	letter-spacing: 3px;	
+}
+
+.contact-link {
+	text-decoration: none;
+	color: #dac7b2;
+}
+
+.home-body {
+	display: flex;
+	justify-content: center;
+	min-height: 100vh;
+}
+
+.spots {
+	display: flex;
+	width: 100%;
+	object-fit: cover;
+	background-color: #dac7b2;
+}
+
+.chloe-smart {
```

Comment:
you don't really need a class for the h1 because there is only ever 1 of it

Reply:
Done